package.json

```diff
@@ -2,13 +2,14 @@
   "name": "uswds",
   "version": "0.9.0",
   "description": "Open source UI components and visual style guide for U.S. government websites",
-  "main": "package.json",
+  "main": "assets/js/components.js",
   "scripts": {
     "build-sass": "node-sass assets/_scss/all.scss > assets/css/uswds.css",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "prepublish": "npm run build-sass build"
```

Comment:
Good catch. I'd had a separate "build" target previously that I'd since taken out. Done.